### ISSUE TITLE
A small question about directly dragging and loading models to mujoco 

### ISSUE THREAD

**Issue by Yingfan99327** [MyoHub/myo_sim#22]
Hi,
Firstly, thanks very much for your great work.
When I try to drag the myoarm.xml, myoleg.xml or myohand.xml to the mujoco.viewer to load the model for visualiztaion on the ubuntu system installed in the virtual machine, the system just stucks and it become really slow for me to click or do any other operations. However, if I load the myoelbow model to the mujoco.viewer the same way, it is ok. So, it maybe a problem related to the computer itself, namely, my computer configuration is not enough to render them in the virtual machine that I need to use a computer with better gpu or cpu because of the high model complexity in the arm or leg model, right?

And could you recommend the lowest computer configuration for the model loading or simulation, my computer's cpu is amd ryzen 7 4800H with radeon graphics and gpu is nvidia geforce gtx 1650. If I install doulbe system instead of using virtual machine, would it be better for simulation? Would apple m1 is ok for the simulation?

Thanks so much for your time.

Wishing you all the best.

**Comment by Vittorio-Caggiano** [comment]
Hi @Yingfan99327 , not sure if you still have this issue as I saw you closed is on MyoSuite https://github.com/MyoHub/myosuite/issues/157#issue-2194787479
In general, I am using an M1 air macbook and I can work on those models in simulate 3.1.2 and 2.3.6 without any problem. Please, let me know if this is still a problem for you and we can look into it more.

**Comment by andreh1111** [comment]
Hi @Yingfan99327, I too sometimes experience some slowdown in simulate (Dell XPS 13 9310), a little trick that works for me is to disable OpenGL effects, especially _Shadow_ and _Reflection_. I hope it can be helpful to you too.

**Comment by Yingfan99327** [comment]
Hi, @Vittorio-Caggiano, thanks so much for your replies and suggestions. Yeah, I just fixed the problems by installing a ubuntu system on my computer instead of using the virtual machine and then the slowdown problem disappears. I also tried to load the model into my m1 computer and it also worked, however, I guess developing in M1 computer may not be a little bit troublesome because of the compatibility I am not sure, but I also kind of notice some issues like runnning mujoco on M1 I guess. Maybe I will use ubuntu more to carry on more work later. Really appreciate your help again and hope you're enjoying a nice day today!😊 

Hi, @andreh1111, thanks very much for your suggestions. I will also check this method if I encounter similar slowdown problems in the future. Really helpful advice! Have a nice day, too!😊
